$NetBSD: patch-doc_Makefile.am,v 1.1 2020/12/06 00:27:51 dbj Exp $

asciidoc 9.0.4 asppears to ignore newline directive in conf file for table of contents
Issue reported upstream at https://github.com/asciidoc/asciidoc-py3/issues/154

--- doc/Makefile.am.orig	2020-11-23 13:49:49.000000000 +0000
+++ doc/Makefile.am
@@ -63,6 +63,7 @@ ASCIIDOC_ARGS = --unsafe -a toc -a data-
 	    false; \
 	fi
 	$(call silent,ADOC  ,$@)TZ=0 $(ASCIIDOC) $(ASCIIDOC_ARGS) --out-file $@ $<
+	@mv $@ $@.bak && tr -d '\r' < $@.bak > $@ # asciidoc 9.0.4 TOC ignored conf file.
 
 all-local: $(ASCIIDOC_MAN_DIST) $(ASCIIDOC_HTML)
 
